tasks/bake.js

```diff
@@ -394,17 +413,29 @@ module.exports = function( grunt ) {
 			else return directory( filePath ) + "/" + includePath;
 		}
 
-		function replaceFile( linebreak, indent, includePath, attributes, filePath, values ) {
+		function processExtraBake( bake, filePath, destFile, values ) {
```

Comment:
Let's leave it here, since this method, if changed to `processBake` is more ambiguous.